translations/readme.md

```diff
@@ -0,0 +1,265 @@
+## Translations
+
+You can help by contributing to translation(s)! Don't worry, even if you totally new to software translations, this guide got you covered!
+
+It's recommended to use [Poedit](https://poedit.net/download/) to work on translation, but using nothing other then [text editor](#working-on-translation-without-additional-tools) is also valid. 
```

Comment:
I believe the intro section should have be a general quick overview of the system.

```suggestion
For translations, GodSVG uses gettext. To edit translations, it's recommended to use [Poedit](https://poedit.net/download/), but using a simple text editor is also a viable option.

[Introduction into the translation system, like what .po and .pot files are]
```